### ISSUE TITLE
unigram_df attribute of Nlpeda module only works if no null values in Dataframe

### ISSUE THREAD

**Issue by ghost** [Elysian01/Data-Purifier#5]
Is this the only way to do it or is there a possibility to maybe deal with null values and/or replace them, so that the unigram_df attribute still works without an error message?

**Comment by Elysian01** [comment]
yes, instead of dropping null rows, we can fill in with some other values, I will update in upcoming versions
